stylesheets/style.css

```diff
@@ -0,0 +1,121 @@
+body {
+    background-color: black;
+    margin: 0px;
+    color: whitesmoke;
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+    grid-template: 1fr 6fr 1fr / 1fr;
+}
+
+header {
+    grid-row: 1 / 2;
+}
+
+footer {
+    grid-row-start: 3 / 4;
+}
+
+header, footer {
+    font-size: 36pt;
+    font-weight: bolder;
+    color: black;
+    background-color: whitesmoke;
+    line-height: 100px;
+    align-items: center;
+    padding-inline-start: 20px;
+    padding-inline-end: 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+nav {
+    display: inline-flex;
+    justify-content: right;
+    font-size: 14pt;
+    font-weight: lighter;
+}
+
+nav a {
+    margin-left: 10px;
+}
+
+a:visited {
+    color: darkgoldenrod
+}
+
+a:link {
+    color:black
+}
+
+p {
+    text-align: center;
+    font-size: 24pt;
+}
+
+#greeting {
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+
+}
+
+#aboutbtn {
+    font-family: Helvetica;
+    text-align: center;
+    width: 15em;
+    height: 3em;
+    margin: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 18pt;
+    background-color: lightblue;
+    border-radius: 20px;
+}
+
+#aboutme {
+    padding: 20vh;
+    display: flex;
+    flex-direction: row;
+}
+
+#aboutme img {
+    max-height: 50vh;
+    max-width: 50vw;
+    padding-right: 5%;
+}
+
+#portfolio {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding-top: 50px;
+    padding-bottom: 20px;
+}
+
+#portfolio p{
+    text-align: left;
+    text-justify: distribute;
+    width: 40vw;
+}
+
+#portfolio a{
+    width: 30vw;
+    height: 30vh;
+    padding-bottom: 20px;
+    justify-items: center;
+}
+
+#portfolio img{
+
```

Comment:
I'd remove the new line here & on line 62 to be consistent with the other style definitions.